pandas/io/pytables.py

```diff
@@ -580,16 +580,40 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         self.close()
 
-    def keys(self) -> List[str]:
+    def keys(self, include: str = "pandas") -> List[str]:
         """
         Return a list of keys corresponding to objects stored in HDFStore.
 
+        Parameters
+        ----------
+
+        .. versionadded:: 1.1.0
```

Comment:
Same comment on other PR - put below include and indent one level (I think will fix CI)

Reply:
fixed